.travis.yml

```diff
@@ -37,30 +37,30 @@ matrix:
   include:
     # 64bit linux builds with GCC
     - os: linux
-      env: TEST_SUITE=testerror
+      env: TEST_SUITE=testerror CONFIGFLAGS=--enable-debug
```

Comment:
I thought at least one build should be without, just as a sanity check that GAP still builds/works without it.